subsys/bluetooth/mesh/sensor.c

```diff
@@ -754,6 +754,10 @@ void sensor_cadence_update(struct bt_mesh_sensor *sensor,
 	sensor->state.fast_pub = new;
 }
 
+#ifndef CONFIG_BT_MESH_SENSOR_USE_LEGACTY_SENSOR_VALUE
+
+#endif /* !CONFIG_BT_MESH_SENSOR_USE_LEGACY_SENSOR_VALUE */
```

Comment:
Is this needed?

Reply:
Heh, no, especially since I typed `LEGACTY`. Sorry, must have missed this when cleaning up